src/main/java/uk/gov/companieshouse/confirmationstatementapi/model/mapping/ConfirmationStatementJsonDaoMapper.java

```diff
@@ -1,14 +1,29 @@
 package uk.gov.companieshouse.confirmationstatementapi.model.mapping;
 
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
 import org.springframework.stereotype.Component;
 import uk.gov.companieshouse.confirmationstatementapi.model.dao.ConfirmationStatementSubmissionDao;
 import uk.gov.companieshouse.confirmationstatementapi.model.json.ConfirmationStatementSubmissionJson;
 
+import java.time.LocalDate;
+
 @Component
 @Mapper(componentModel = "spring")
 public interface ConfirmationStatementJsonDaoMapper {
 
+      @Mapping(source = "data.madeUpToDate", target = "data.madeUpToDate", qualifiedByName = "localDate")
       ConfirmationStatementSubmissionJson daoToJson(ConfirmationStatementSubmissionDao confirmationStatementSubmissionDao);
+
+      @Mapping(source = "data.madeUpToDate", target = "data.madeUpToDate", qualifiedByName = "localDate")
       ConfirmationStatementSubmissionDao jsonToDao(ConfirmationStatementSubmissionJson confirmationStatementSubmissionJson);
+
+      @Named("localDate")
```

Comment:
LocalDate doesn't have a default constructor so the mapping as it was wouldn't map the date, it would come out as null.  So a custom mapper was needed for LocalDates